<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PupguiInstallDialog</class>
 <widget class="QDialog" name="PupguiInstallDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>280</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>280</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>320</width>
    <height>280</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Install Compatibility Tool</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>10</y>
     <width>301</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item>
     <widget class="QLabel" name="lblCompatTool">
      <property name="text">
       <string>Compatibility tool:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboCompatTool">
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblCompatToolVersion">
      <property name="text">
       <string>Version:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboCompatToolVersion"/>
    </item>
    <item>
     <widget class="QLabel" name="lblTxtDescription">
      <property name="text">
       <string>Description:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="txtDescription">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>95</height>
       </size>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="button_box">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnInfo">
        <property name="text">
         <string>Info</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnInstall">
        <property name="text">
         <string>Install</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
